networking_aci/plugins/ml2/drivers/mech_aci/agent/aci_agent.py

```diff
@@ -186,9 +188,34 @@ def _handle_sigterm(self, signum, frame):
 
     # End Agent mechanics
 
-    def _run_nullroute_sync(self):
+    def _run_non_epg_syncloop(self):
         ctx = context.get_admin_context_without_session()
-        self.sync_nullroutes(ctx)
+        LOG.info("Starting periodic non-epg syncloop")
+        if CONF.ml2_aci.enable_az_aware_subnet_routes_sync:
+            try:
+                self.sync_az_aware_subnet_routes(ctx)
+            except Exception as e:
+                LOG.exception("Sync of AZ aware subnet routes failed: %s %s",
+                              e.__class__.__name__, e)
+        else:
+            LOG.info("AZ aware subnet routes syncloop is currently disabled")
+
+        if CONF.ml2_aci.enable_nullroute_sync:
+            try:
+                self.sync_nullroutes(ctx)
```

Comment:
Should we maybe go with the networking-ccloud streamlined naming of aggregates, even though here in aci-land they are nullroutes?

Reply:
I don't know, we've discussed and planned them out as nullroutes so far and it's what they're doing, so I'd stick with the naming.